makefile

```diff
@@ -74,8 +74,22 @@ all: $(OUTPUTDIR) libJyNI libJyNI-Loader JyNI
 debug: CFLAGS += -g
 debug: all
 
-tests:
-	@echo 'Tests have not been implemented yet'
+clean:
+	rm -rf $(JYNIBIN)
+	rm -f ./JyNI-C/src/*.o
+	rm -f ./JyNI-C/src/Python/*.o
+	rm -f ./JyNI-C/src/Objects/*.o
+	rm -f ./JyNI-C/src/Modules/*.o
+	rm -f ./JyNI-Loader/JyNILoader.o
+
+tests: build-tests run-tests
+
+build-tests:
+	@echo 'building tests is not fully supported yet, this will either not work or try to install the demo extension as an actuall extension'
+	python ./DemoExtension/setup.py install
```

Comment:
What would this actually do? I suspect this might install it to system's CPython. If that is the case I think we should better keep it locally in the JyNI folder. Would ``python ./DemoExtension/setup.py build`` be sufficient here? I think that's what I usually call to create DemoExtension.so (or DemoExtension.pyd).

Reply:
Sorry I had forgotten about that, I didn't realise `python setup.py build` was a thing. It seems to work in eclipse as well as the command line, I'll update the branch/pull request.